samples/msgext-northwind-inventory-csharp/Bot/SearchBot.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Schema.Teams;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using System;
+using msgext_northwind_inventory_csharp.MessageExtensions;
+using Newtonsoft.Json;
+using msgext_northwind_inventory_csharp.Handlers;
+
+namespace msgext_northwind_inventory_csharp.Bots
```

Comment:
Let's update the namespace with a friendlier name

```suggestion
namespace NorthwindInventory.Bots
```

Reply:
fixed